src/CoreWCF.WebHttp/src/CoreWCF/Channels/HttpStreamMessage.cs

```diff
@@ -0,0 +1,220 @@
+        }
+
+        public override bool IsEmpty
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override bool IsFault
+        {
+            get { return false; }
+        }
+
+        public override MessageProperties Properties
+        {
+            get
+            {
+                if (IsDisposed)
+                {
+                    throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(CreateDisposedException());
+                }
+                return _properties;
+            }
+        }
+
+        public override MessageVersion Version
+        {
+            get
+            {
+                if (IsDisposed)
+                {
+                    throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(CreateDisposedException());
+                }
+                return MessageVersion.None;
+            }
+        }
+
+        protected override void OnBodyToString(XmlDictionaryWriter writer)
+        {
+            if (_bodyWriter.IsBuffered)
+            {
+                _bodyWriter.WriteBodyContents(writer);
+            }
+            else
+            {
+                writer.WriteString(SR.Format(SR.MessageBodyIsStream));
+            }
+        }
+
+        protected override void OnClose()
+        {
+            Exception ex = null;
+            try
+            {
+                base.OnClose();
+            }
+            catch (Exception e)
+            {
+                if (Fx.IsFatal(e))
+                {
+                    throw;
+                }
+                ex = e;
+            }
+
+            try
+            {
+                if (_properties != null)
+                {
+                    _properties.Dispose();
+                }
+            }
+            catch (Exception e)
+            {
+                if (Fx.IsFatal(e))
+                {
+                    throw;
+                }
+                if (ex == null)
+                {
+                    ex = e;
+                }
+            }
+
+            if (ex != null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(ex);
+            }
+
+            _bodyWriter = null;
+        }
+
+        protected override MessageBuffer OnCreateBufferedCopy(int maxBufferSize)
+        {
+            BodyWriter bufferedBodyWriter;
+            if (_bodyWriter.IsBuffered)
+            {
+                bufferedBodyWriter = _bodyWriter;
+            }
+            else
+            {
+                bufferedBodyWriter = _bodyWriter.CreateBufferedCopy(maxBufferSize);
+            }
+
+            return new HttpStreamMessageBuffer(Headers, new MessageProperties(Properties), bufferedBodyWriter);
+        }
+
+        protected override void OnWriteBodyContents(XmlDictionaryWriter writer)
+        {
+            _bodyWriter.WriteBodyContents(writer);
+        }
+
+        Exception CreateDisposedException()
+        {
```

Comment:
Accessibility modifier

Reply:
Addressed in fa64f8fd703f0284110f2f879ed6896bf89d8ac6.